Update AI resource skills (#9696)
rilldata/rill: runtime/ai/instructions/data/resources/rillyaml.md

Common values are `duckdb` or `clickhouse`. If not specified, Rill initializes a managed DuckDB database and uses it as the default OLAP connector. 

### Default AI connector

> The `ai_connector` property selects the connector powering AI features (developer agent, AI charts, chat). Point it at a project AI connector such as `openai`, `claude`, or `gemini`. Defaults to Rill's built-in AI service.

Reviewer: ```suggestion
The `ai_connector` property selects the connector powering AI features (developer agent, AI charts, chat). Point it to an AI connector defined in the project, such as `openai`, `claude`, or `gemini`. Defaults to Rill's built-in AI service.
```
